### PR TITLE
Add Hex Code

### PR DESCRIPTION
**Pull request by Pixelatedza**
Adding Hexadecimal Code of current character.

### CODE REVIEW

File: show_char_code.py [hunk 1]
```diff
@@ -11,4 +11,4 @@ def on_activated(self, view):
 
     def show_char_code(self, view):
         selected = view.substr(view.sel()[0].a)
-        view.set_status('charcode', "Char Code: {0}".format(ord(selected)))
```
Comment (borislubimov) [inline show_char_code.py:14]: Hi, thanks for PR! I think it's better to use `{0:#x}` to display hex code. Also I noticed that in [fork of my repo](https://github.com/onelittlehope/ShowCharacterCode/blob/master/show_char_code.py) already added hex and byte codes. Maybe it's worth to do here the same so these changes would be available through Package Control. The only thing I'm not sure is the name "BYTE". Is that name correct for such character representation?

Comment (Pixelatedza) [reply show_char_code.py:14]: I liked the cleaner look, but I agree that #x is a better approach.

I saw the fork doing something similar. I didn't want to clutter up the bottom too much and thought binary might be a bit excessive. Maybe making it an option that can be turned on/off?

**Comment by Pixelatedza** [comment]
I've added the code for binary representation as well.

**Comment by borislubimov** [comment]
Binary representation indeed a bit excessive, so I replaced it with byte representation which is more practical

**Pull request merged**